features/currency/handling_different_currencies_on_multiple_channels.feature

```diff
@@ -0,0 +1,30 @@
+@currencies
+Feature: Handling different currencies on multiple channels
+    In order to see right prices
+    As a Customer
+    I want to browse channels only with valid currency
+
+    Background:
+        Given the store operates on a channel named "Web"
+        And that channel allows for "EUR", "USD" and "GBP" currencies
```

Comment:
`And that channel allows for "EUR", "USD" and "GBP" currencies` -> `And that channel allows for paying in euro, american dollars and british pounds currencies`?